src/test/java/org/apache/maven/plugins/shade/DefaultShaderTest.java

```diff
@@ -364,55 +359,162 @@ private static DefaultShader newShader()
     {
         DefaultShader s = new DefaultShader();
 
-        s.enableLogging( new ConsoleLogger( Logger.LEVEL_INFO, "TEST" ) );
-
         return s;
     }
 
-    private static class MockLogger extends AbstractLogger
+    private static class MockLogger extends MarkerIgnoringBase
```

Comment:
Is there simpler way to achieve this? SLF4J logger that collects debug and warn messages...